### PR TITLE
[MAP] Church Rework

### PR DESCRIPTION
**Pull request by hyperioo**
## About The Pull Request

Merging of cl's and done's suggestions for the map community event. 
z-level 1 is cl's, z-levels 2 and 3 are done's. It does not look exactly like their suggestions because I had to map some stuff to make them compatible.

* New wooden floor tiles that are not painful to look at (courtesy of infrared baron)

* Bar is unchanged.

* Z-level 1

church_rework_1end

* Z-level 2

church_rework_2end

* Z-level 3

church_rework_3end

* Z-level 4

church_rework_4end

## Why It's Good For The Game

Hopefully a better NeoTheology experience.

## Changelog
:cl: Hyperio
add: Church map rework
/:cl:

### CODE REVIEW

**Comment by zombiemimic** [comment]
out of curiosity, when do you plan to add in club changes?

File: code/game/turfs/flooring/flooring_premade.dm [hunk 1]
```diff
@@ -59,7 +59,9 @@
 	icon_state = "wood"
 	initial_flooring = /decl/flooring/wood
 
-
+/turf/simulated/floor/wood/Initialize()
+	icon_state = pick("wood", "wood1", "wood2")
+	return ..()
```
Comment (TheShown911) [inline code/game/turfs/flooring/flooring_premade.dm:64]: This randomizes how the wood looks on every tile? Does it fit together nicely?

Comment (Firefox13) [reply code/game/turfs/flooring/flooring_premade.dm:64]: I've downloaded and checked few times(restarted the round) branch to see the tiles and they were fine to me each time.

**Review by Firefox13 — APPROVED** [review]

**Comment by hyperioo** [comment]
@zombiemimic Nothing is currently planned for Club so it will be staying as it is in the foreseable future, to quote the message in news channel that announced the results: "All map edits of the Club did not stand out impressively and none really seemed to satisfy us."

**Pull request merged**